controllers/v1/retrospective.go

```diff
@@ -126,3 +147,28 @@ func (ctrl RetrospectiveController) Create(c *gin.Context) {
 
 	c.JSON(status, retro)
 }
+
+// Update ...
+func (ctrl RetrospectiveController) Update(c *gin.Context) {
+	userID, _ := c.Get("userID")
+	var err error
+	retrospectiveData := retrospectiveSerializers.RetrospectiveUpdateSerializer{}
+	if err = c.BindJSON(&retrospectiveData); err != nil {
+		c.AbortWithStatusJSON(http.StatusBadRequest, gin.H{"error": "invalid request data"})
```

Comment:
Let's also log the actual error in the bind method here.

Reply:
I think we should implement logging in the project together.